### ISSUE TITLE
Is this project dead? 

### ISSUE THREAD

**Issue by zefoo** [milio48/serveo#24]
I noticed http://serveo.net is down - and has been down. Did this project die? Or just hosting die? I still use serveo but couldn't get to the docs... thanks.

**Comment by talwat** [comment]
It's still down a year later, it's probably dead.

**Comment by 702260** [comment]
yes i am not able to access http://servero.net. also have been trying ssh port forwarding from this site. not working Dec 2022.

**Comment by stevenventimiglia** [comment]
Site redirects to browser "safety" spam.  EOL.

**Comment by chrissy0** [comment]
Seems to be up again.

**Comment by daumienebi** [comment]
Its up again

**Comment by Parking-Master** [comment]
Anddd now it's down.

**Comment by terminal-cs** [comment]
> Anddd now it's down.

yeah i saw this too, literally worked yesterday. Anyone else know what this could be?

**Comment by Parking-Master** [comment]
@terminal-cs I was in the middle of a project and I thought "finally! A service that's free and works" but I guess I got my hopes up too high.

**Comment by terminal-cs** [comment]
> @terminal-cs I was in the middle of a project and I thought "finally! A service that's free and works" but I guess I got my hopes up too high.

you could try ngrok or http://localhost.run/ (if that works for you)

**Comment by Parking-Master** [comment]
Yeah but the reason I use it is because it's probably one of the only services that serve free subdomains, I used to use the services you linked and those don't have subdomains (at least for free)

**Comment by Parking-Master** [comment]
Yes it's back up! Guess it isn't dead after all lol

**Comment by souvikmishra** [comment]
Well its down again 😢

**Comment by Parking-Master** [comment]
@souvikmishra not for me. Try doing a Shift+Reload, it should be up

**Comment by Parking-Master** [comment]
It's reported down since 10AM this morning (EST)

**Comment by AroliSG** [comment]
down once again!

**Comment by phoneticallySAARTHaK** [comment]
It's up now

**Comment by Martzy303** [comment]
It goes up and down, have used it for a couple of months, mostly is up.

**Comment by Parking-Master** [comment]
It's been down for 3+ days. Hopefully it comes back soon.

**Comment by Martzy303** [comment]
It's up!

**Comment by terminal-cs** [comment]
> It's up!

Yay!

**Comment by Parking-Master** [comment]
Been down for days again
